src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures/custom_fixtures.yaml

```
@@ -13,10 +13,17 @@ sylius_fixtures:
                     options:
                         amount: 3
 
-                order:
+                new_order:
+                    name: order
                     options:
-                        amount: 20
+                        amount: 6
                         channel: 'FASHION_WEB'
+                fulfilled_order:
+                    name: order
+                    options:
+                        amount: 14
```

```suggestion
                        amount: 12
```
to have 60% of fulfilled orders 🎉